README.md

```diff
@@ -187,65 +298,118 @@ Translations in Django templates use one of two template tags, `{% t %}` and
 strings themselves can be constant, or they can contain variables which can be
 resolved either from the parameters of the template tags or the context of the
 template. The difference between the two tags will be addressed later, under
-[XML escaping](#XML-escaping).
+[XML escaping](#XML-Escaping).
 
 ```html
 <p>{% t "This is a great sentence." %}</p>
 <h2>{% t "Welcome, {username}" username=user.name %}</h2>
 <pre>{% t snippet.code  %}</pre>
 ```
 
-##### Inline and Block syntax
+Again, ICU Message Format is supported, so you can use the same string syntax as in Django templates, and pass all variables as named arguments.
```

Comment:
The examples are from python code, not template code

Reply:
Done